core/src/events.rs

```diff
@@ -136,7 +136,7 @@ impl CollisionData {
 
     /// Returns the normal vector pointing toward this entity and away from the other entity
     #[must_use]
-    pub fn normals(&self) -> &[Vec2] {
+    pub fn normals(&self) -> &[Vec3] {
         &self.normals
     }
 }
```

Comment:
This is an unrelated change. Please revert and open another PR.

Also, note that this is a breaking change. I understand, it is due to the mistake of having exposed 2d vectors instead of the 3d ones. But nevertheless, we shall try to improve it without breaking the API.